requirements.txt

```diff
@@ -13,4 +13,6 @@ playwright==1.43.0
 google-api-python-client==2.100.0
 wikipedia==1.4.0
 python-gitlab==4.5.0
-beautifulsoup4
\ No newline at end of file
+alita-sdk==0.1.4
```

Comment:
remove this - as it will cause circular dependencies. Alita-SDK rely on application-tools

Reply:
The dep have been removed. The llm facilities in a short format have been added to app tools repo instead